tuf/roledb.py

```diff
@@ -963,7 +971,10 @@ def get_delegated_rolenames(rolename, repository_name='default'):
   delegated_roles = []
 
   for delegated_role in roleinfo['delegations']['roles']:
-    delegated_roles.append(delegated_role['name'])
+    if 'succinct_hash_delegations' in delegated_role:
+      delegated_roles.append(delegated_role['succinct_hash_delegations']['bin_name_prefix'])
```

Comment:
Is the prefix enough here, or do we need the full role name?

Reply:
I'm not sure what the expected behavior here would be. A single succinct delegation in the metadata corresponds to many actual delegations, so should this function return names of the delegated roles in the metadata file (what it currently does), or the names of all roles delegated to by the current role (including the full names of all succinct delegations)?

Either way, I'll be sure to document it in the docstring.